src/_premake_init.lua

```diff
@@ -241,6 +241,24 @@
 	}
 
 	api.register {
+		name = "debugger",
+		scope = "config",
+		kind = "string",
+		allowed = {
+			"VisualStudio",
+			"GDB",
+			"LLDB",
+		}
+	}
```

Comment:
This is a high level debugger selection. It is intended to generate project bindings for the chosen debugger.